### PR TITLE
i18n + favicon

### PR DESCRIPTION
**Pull request by olivierteboul**

### CODE REVIEW

File: icubam/backoffice/templates/base.html [hunk 1]
```diff
@@ -114,46 +114,46 @@
...
-                <p>Map</p>
+                <p>{{_("Map")}}</p>
               </a>
             </li>
             {% if this_user.is_admin %}
             <li class="nav-item">
               <a href="operational-dashboard" class="nav-link">
                 <i class="nav-icon fas fa-chart-pie"></i>
-                <p>Operational dashboard</p>
+                <p>{{_("Operational dashboard")}}</p>
```
Comment (dulacarnold) [inline icubam/backoffice/templates/base.html:144]: dashboard -> Dashboard

File: icubam/backoffice/templates/base.html [hunk 2]
```diff
@@ -164,46 +164,24 @@
...
-        <p>Sidebar content</p>
-      </div>
-    </aside>
-    <!-- /.control-sidebar -->
 
-    <!-- Main Footer -->
     <footer class="main-footer">
-      <!-- To the right -->
       <div class="float-right d-none d-sm-inline">
         Where is the ICU?
```
Comment (dulacarnold) [inline icubam/backoffice/templates/base.html:174]: Glad you kept this :)

File: icubam/backoffice/translations/fr.csv [hunk 3]
```diff
@@ -23,6 +23,7 @@
 "n_covid_healed","Guéris"
 "n_covid_refused","Refusés"
 "n_covid_transfered","Transférés"
+"icu (update link)","Réa (lien MAJ)"
```
Comment (dulacarnold) [inline icubam/backoffice/translations/fr.csv:26]: icu -> ICU.

File: icubam/backoffice/translations/fr.csv [hunk 4]
```diff
@@ -35,4 +36,63 @@
...
+"is_active","Actif"
+"expiration_date","Expire"
+"access_type","type d'accès"
+"map","Carte"
+"db/bedcounts","Stats"
+"scheduled","Prochain envoi"
+"attempts /{}","Essais /{}"
+"first sent","Premier essai"
+"url","lien"
+"telephone","Téléphone"
```
Comment (dulacarnold) [inline icubam/backoffice/translations/fr.csv:73]: Maybe we should either have everything in lower or all Starting Upper?

File: icubam/backoffice/templates/list.html [hunk 5]
```diff
@@ -16,27 +16,26 @@
...
-         <h1>{{objtype}}</h1>
-       </div>
-     </div>
-   </div><!-- /.container-fluid -->
-  </section>
+    <section class="content-header">
+      <div class="container-fluid">
+        <div class="row mb-2">
+          <div class="col-sm-6">
+            <h1>{{_(objtype)}}</h1>
```
Comment (dulacarnold) [inline icubam/backoffice/templates/list.html:23]: Are we sure to have sufficient translations for all objtypes?

File: icubam/backoffice/templates/base.html (hunk 2 above)
Comment (olivierteboul) [reply icubam/backoffice/templates/base.html:174]: hhaha. i am opened to suggestions

File: icubam/backoffice/templates/list.html (hunk 5 above)
Comment (olivierteboul) [reply icubam/backoffice/templates/list.html:23]: If not, the strings without translations are kept unchanged.

File: icubam/backoffice/templates/base.html (hunk 1 above)
Comment (olivierteboul) [reply icubam/backoffice/templates/base.html:144]: done

File: icubam/backoffice/translations/fr.csv (hunk 3 above)
Comment (olivierteboul) [reply icubam/backoffice/translations/fr.csv:26]: done

File: icubam/backoffice/translations/fr.csv (hunk 4 above)
Comment (olivierteboul) [reply icubam/backoffice/translations/fr.csv:73]: i would be in favor of a en.csv, to make sure things are always the way we want.

**Pull request merged**